src/Solutions/Folds.idr

```diff
@@ -85,10 +81,7 @@ bindTR xs f = go Lin xs
         go sx (x :: xs) = go (sx <>< f x) xs
 
 joinTR : List (List a) -> List a
-joinTR = go Lin
-  where go : SnocList a -> List (List a) -> List a
-        go sx []        = sx <>> Nil
-        go sx (x :: xs) = go (sx <>< x) xs
+joinTR xss = bindTR xss id
```

Comment:
While this is a tail-recursive implementation of `join`, this is not what this exercise is about. It's about teaching how to manually write tail-recursive functions not about the connection between `join` and `bind`. I won't accept this change, but I'll accept it as a second - better - solution together with an explaining comment.

Reply:
Done: https://github.com/stefan-hoeck/idris2-tutorial/pull/75/commits/5bf0eef53bcdca1aca0009fc49964eed4ee682a5